Source/OxyPlot.SkiaSharp.Avalonia/OxyPlotModule.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Avalonia.Metadata;
+
+[assembly: XmlnsDefinition("http://oxyplot.org/avalonia", "OxyPlot.Avalonia")]
+[assembly: XmlnsDefinition("http://oxyplot.org/skiasharp/avalonia", "OxyPlot.SkiaSharp.Avalonia")]
+[assembly: XmlnsDefinition("http://oxyplot.org/skiasharp/avalonia/doublebuffered", "OxyPlot.SkiaSharp.Avalonia.DoubleBuffered")]
```

Comment:
Is there an advantage to this? Obviously it makes sense from an XML stand-point, but are there advantages over using the `clr-namespace` or `using` notations supported by Avalonia in the XAML?

Reply:
No idea, I just copied and adapted from here:
https://github.com/oxyplot/oxyplot-avalonia/blob/9913a97d79a75585abd0ee5b417739ac52bca4c6/Source/OxyPlot.Avalonia/OxyPlotModule.cs#L4

Is there a better way to do this? 🤔

Reply:
I don't know; I prefer talking about the CLR namespace rather than the XML namespace, but I believe this was how it was traditionally done in WPF and I don't know if there are benefits. I'm happy with it as-is, but was wondering if you'd changed the xmlns in the axaml because it has some benefit.